### PR TITLE
Add notes about the field declarations

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
#### Summary

Add notes about the field declarations.
#### Motivation

See https://stackoverflow.com/questions/72525292/why-field-declarations-dont-use-keywords-in-javascript-es6
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/classes/index.md [hunk 1]
```diff
@@ -259,6 +259,8 @@ class Rectangle {
 }
 ```
 
+We don't use any keywords like `let`,`const` and `var` before the fields.
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/classes/index.md:262]:
```suggestion
We don't need statements like `let`,`const`, and `var` before fields.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks a lot, I've rewritten it a bit. What do you think?

**Comment by awxiaoxian2020** [comment]
> Thanks a lot, I've rewritten it a bit. What do you think?

It's OK! Thanks!

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Comment by awxiaoxian2020** [comment]
@teoli2003 Thanks for merging!
I want to know when it will be deployed and should I sync to the `translated-content` immediately? I want to know the workflow :)

File: files/en-us/web/javascript/reference/classes/index.md [hunk 2]
```diff
@@ -259,6 +259,8 @@ class Rectangle {
 }
 ```
 
+We don't need statements like `let`,`const`, and `var` before fields.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/classes/index.md:262]: Errr... Not a huge deal, but I'd like to mention that `let` / `const` are not statements but keywords, and `let x` is a declaration, not a statement either. See #16307 also

**Comment by teoli2003** [comment]
> @teoli2003 Thanks for merging! I want to know when it will be deployed and should I sync to the `translated-content` immediately? I want to know the workflow :)

The production website is usually built once a day; read this [section](https://github.com/mdn/content#when-will-my-change-show-up-on-the-production-mdn-site) for more information.

You can already submit the PR to mdn/translated-content now.

File: files/en-us/web/javascript/reference/classes/index.md (hunk 2 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/classes/index.md:262]: OK, added!

**Pull request merged**